src/SDK/Language/CLI.php

```diff
@@ -4,6 +4,39 @@
 
 class CLI extends PHP {
 
+    /**
+     * @var array
+     */
+    protected $params = [
+        'composerVendor' => 'vendor-name',
+        'composerPackage' => 'package-name',
+        'executableName' => 'executable',
+        'logo' => 'logo'
```

Comment:
Am empty string would be a better default here.

Reply:
Okay I will do that 👍